### PR TITLE
Refine ParallelExecutor English Doc for 2.0RC

### PR DESCRIPTION
**Pull request by zhhsplendid**
### PR types

Others
### PR changes

Docs
### Describe

As the title, screenshot:
image

### CODE REVIEW

File: python/paddle/fluid/parallel_executor.py [hunk 1]
```diff
@@ -205,11 +206,11 @@ def run(self, fetch_list, feed=None, feed_dict=None, return_numpy=True):
         fetch_list.
 
         Args:
-            fetch_list(list): This parameter represents the variables that need to be returned
+            fetch_list(list): This parameter represents the tensors that need to be returned
```
Comment (liym27) [inline python/paddle/fluid/parallel_executor.py]: tensors -> **T**ensors, the same below

File: python/paddle/fluid/parallel_executor.py [hunk 2]
```diff
@@ -241,19 +242,20 @@ def run(self, fetch_list, feed=None, feed_dict=None, return_numpy=True):
...
               import numpy
               import os
 
               use_cuda = True
-              place = fluid.CUDAPlace(0) if use_cuda else fluid.CPUPlace()
+              paddle.enable_static()
+              place = paddle.CUDAPlace(0) if use_cuda else paddle.CPUPlace()
 
               # NOTE: If you use CPU to run the program, you need
               # to specify the CPU_NUM, otherwise, fluid will use
```
Comment (liym27) [inline python/paddle/fluid/parallel_executor.py]: fluid -> Paddle

Comment (zhhsplendid) [reply python/paddle/fluid/parallel_executor.py]: Done

**Review by Aurelius84 — APPROVED** [review]

**Review by liym27 — APPROVED** [review]

**Review by dingjiaweiww — APPROVED** [review]

**Review by Xreki — APPROVED** [review]

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**